ByBit.Net/Clients/SpotApi/v3/BybitClientSpotApiTradingV3.cs

```diff
@@ -236,7 +237,8 @@ public async Task<WebCallResult<IEnumerable<BybitRepayRecord>>> GetRepayRecordsA
             parameters.AddOptionalParameter("limit", limit);
             parameters.AddOptionalParameter("recvWindow", receiveWindow?.ToString(CultureInfo.InvariantCulture) ?? _baseClient.ClientOptions.ReceiveWindow.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
 
-            return await _baseClient.SendRequestAsync<IEnumerable<BybitRepayRecord>>(_baseClient.GetUrl("spot/v3/private/cross-margin-repay-history"), HttpMethod.Get, ct, parameters, true).ConfigureAwait(false);
+            var result = await _baseClient.SendRequestAsync<BybitRepayRecordWrapper>(_baseClient.GetUrl("spot/v3/private/cross-margin-repay-history"), HttpMethod.Get, ct, parameters, true).ConfigureAwait(false);
+            return result.As(result.Data.Repays);
```

Comment:
This has the same issue as the earlier PR, result.Data is null when the call fails